engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -739,6 +871,35 @@ else if (propertyName.equals(entry.getKey())) {
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}/properties")
```

Comment:
The code to extract properties from operator and module is same, I have made it as a utility function. This top level function will look for the given name in modules list and use the utility function to print the property.